index.html

```diff
@@ -298,6 +297,7 @@ <h1 class="primary-text">Contact me</h1>
                     <textarea cols="30" placeholder="Please write your message" id="message" rows="10" class="input-type-textarea"
                         name="message" maxlength="500" required></textarea>
                     <input type="submit" value="Get in Touch" class="get-in-touch" id="button-submit"/>
+                    <div class="error-output" id="error-output"></div>
```

Comment:
[OPTIONAL] I think is a good idea to avoid the user having to scroll down to notice the error. :eyes: :eyes: You could try to show the error message next to the submit button. :+1: :smiley: :smiley: